### PR TITLE
Task/sdk 2502/add android13 support

### PR DESCRIPTION
**Pull request by shivamsharma2710**
Changes:

- Upgraded plugin specific android platform to target android 13
- Updated the CleverTap Android SDK dependency with the v4.7.2
- Exposed methods for android 13 handling.

### CODE REVIEW

**Review by akashvercetti — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -15,14 +15,14 @@ buildscript {
...
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 31
-        versionCode 93
-        versionName "0.9.3"
+        minSdkVersion 19
+        targetSdkVersion 33
+        versionCode 95
+        versionName "0.9.5"
```
Comment (darshanclevertap) [inline android/build.gradle:25]: @shivamsharma2710 We should update this to version 1.0.0

File: Example/android/app/build.gradle [hunk 2]
```diff
@@ -216,7 +216,7 @@ dependencies {
 
 
     //clevertap
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.6'
+    implementation 'com.clevertap.android:clevertap-android-sdk:4.7.2'
     implementation "com.clevertap.android:push-templates:1.0.5"
```
Comment (darshanclevertap) [inline Example/android/app/build.gradle:220]: @shivamsharma2710 update push templates sdk to v1.0.7

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clevertap-react-native",
-  "version": "0.9.4",
+  "version": "0.9.5",
```
Comment (darshanclevertap) [inline package.json:3]: @shivamsharma2710 Update to v1.0.0

**Review by darshanclevertap — COMMENTED** [review]
@shivamsharma2710 left comments for you

File: android/build.gradle (hunk 1 above)
Comment (shivamsharma2710) [reply android/build.gradle]: this is done. please check.

File: Example/android/app/build.gradle (hunk 2 above)
Comment (shivamsharma2710) [reply Example/android/app/build.gradle]: this is done. please check.

File: package.json (hunk 3 above)
Comment (shivamsharma2710) [reply package.json]: this is done. please check.

**Review by darshanclevertap — APPROVED** [review]

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Pull request merged**